src/registrar/admin.py

```diff
@@ -2656,6 +2656,32 @@ class Meta:
         fields = "__all__"
 
 
+class DomainGroupResource(resources.ModelResource):
+    """defines how each field in the referenced model should be mapped to the corresponding fields in the
```

Comment:
@dave-kennedy-ecs should we add these for new models? (I am thinking not) This would require additional work to get them imported correct?

Reply:
what would determine if something needs to be included in the export/import tables? For instance, if we add a new value to domain requests/domain info that is an fk to another table, is that when we would need to update the import/export feature so that we don't break anything?

Reply:
Yes, that is one condition that would require us to update import/export - anything new tables which have fk from a current table in the import/export set.
And, generally, any data that needs to be replicated in an export/import scenario would need to update the import/export scripts